packages/ember-runtime/lib/mixins/array.js

```diff
@@ -573,3 +583,55 @@ export default Mixin.create(Enumerable, {
     return this.__each;
   }).volatile()
 });
+
+if (isEnabled('ember-runtime-enumerable-includes')) {
+  ArrayMixin.reopen({
+    /**
+      Returns `true` if the passed object can be found in the array.
+      This method is a Polyfill for ES 2016 Array.includes.
+      If no `startAt` argument is given, the starting location to
+      search is 0. If it's negative, searches from the index of
+      `this.length + startAt` by asc.
+      ```javascript
+      [1, 2, 3].includes(2);     // true
+      [1, 2, 3].includes(4);     // false
+      [1, 2, 3].includes(3, 2);  // true
+      [1, 2, 3].includes(3, 3);  // false
+      [1, 2, 3].includes(3, -1); // true
+      [1, 2, 3].includes(1, -1); // false
+      [1, 2, 3].includes(1, -4); // true
+      [1, 2, NaN].includes(NaN); // true
+      ```
+      @method includes
+      @param {Object} obj The object to search for.
+      @param {Number} startAt optional starting location to search, default 0
+      @return {Boolean} `true` if object is found in the array.
+      @public
+    */
+    includes(obj, startAt) {
+      var len = get(this, 'length');
+      var idx, currentObj;
+
+      if (Ember.isNone(startAt)) {
```

Comment:
Anyway, explicit null check is unnecessary as we only iterating on `idx = startAt`. I removed it.